lib/livebook/intellisense/identifier_matcher.ex

```diff
@@ -21,6 +21,7 @@ defmodule Livebook.Intellisense.IdentifierMatcher do
   @type identifier_item ::
           {:variable, name(), value()}
           | {:map_field, name(), value()}
+          | {:in_struct_field, struct_name :: display_name(), name(), default :: value()}
```

Comment:
I'd keep the actual module here and convert to string as in #798 (feel free to copy the conversion and I will rebase that PR later)

```suggestion
          | {:in_struct_field, module(), name(), default :: value()}
```